tests/test_geo_opt.py

```diff
@@ -0,0 +1,126 @@
+"""Test geometry optimisation."""
+
+from pathlib import Path
+
+try:
+    from ase.filters import UnitCellFilter
+except ImportError:
+    from ase.constraints import UnitCellFilter
+import pytest
+
+from janus_core.geo_opt import optimize
+from janus_core.single_point import SinglePoint
+
+
+def test_optimize():
+    """Test optimizing geometry using MACE."""
+    data_path = Path(Path(__file__).parent, "data", "NaCl.cif")
+    model_path = Path(Path(__file__).parent, "models", "mace_mp_small.model")
+    single_point = SinglePoint(
+        system=data_path, architecture="mace", model_paths=model_path
+    )
+
+    init_energy = single_point.run_single_point("energy")["energy"]
+
+    atoms = optimize(single_point.sys)
+
+    assert atoms.get_potential_energy() < init_energy
+    assert atoms.get_potential_energy() == pytest.approx(-27.046359959669214)
+
+
+@pytest.mark.slow
+def test_fmax():
+    """Test changing fmax."""
+    data_path = Path(Path(__file__).parent, "data", "NaCl.cif")
+    model_path = Path(Path(__file__).parent, "models", "mace_mp_small.model")
```

Comment:
Looking at this, we probably want top-level globals pointing to `data` and `models` folders rather than this. But see above

```python
DATA_PATH = Path(__file__).parent / "data"
DATA_PATH = Path(__file__).parent / "models"
```